### PR TITLE
Upgrading Igor-oes from OES-1.30.X to OES-1.33.X

### PR DESCRIPTION
**Pull request by emanipravallika**
https://devopsmx.atlassian.net/browse/OP-22085 : **ParentJira**
https://devopsmx.atlassian.net/browse/OP-22196 : **JIra :**
**Testing :**

Compile/Build got success, All TestCases passed, No new TestCases failed.
igor-monitor-plugins: Results: SUCCESS (6 tests, 6 successes, 0 failures, 0 skipped)
igor-core: Results: SUCCESS (43 tests, 43 successes, 0 failures, 0 skipped)
igor-monitor-artifactory: Results: SUCCESS (9 tests, 9 successes, 0 failures, 0 skipped)
igor-monitor-travis: Results: SUCCESS (106 tests, 106 successes, 0 failures, 0 skipped)
igor-web: Results: SUCCESS (350 tests, 350 successes, 0 failures, 0 skipped)
Deployed docker image in dev-spin-cve-cluster cvetarget ns

**UseCase tested** : Configured jenkins in cvetarget ns and tested the jenkins pipeline stage. working as expected.
verified igor logs, looks clean
**Note :**
All commits covered - verified from sheet.
fiatVersion=1-0-SNAPSHOT & korkVersion=1.33.2-nemesis-SNAPSHOT

### CODE REVIEW

File: igor-web/igor-web.gradle [hunk 1]
```diff
@@ -4,93 +4,134 @@ mainClassName = 'com.netflix.spinnaker.igor.Main'
...
+  implementation "jakarta.validation:jakarta.validation-api:3.0.2"
+  implementation "com.fasterxml.jackson.module:jackson-module-jaxb-annotations"
+  implementation("io.netty:netty-codec-http:4.1.86.Final") {
+    force = true
+  }
+  testImplementation("io.netty:netty-codec-http:4.1.86.Final") { force = true }
+  runtimeOnly "io.spinnaker.kork:kork-runtime"
+  implementation "io.spinnaker.kork:kork-retrofit"
+
+  runtimeOnly "io.spinnaker.kork:kork-runtime"
```
Comment (aman-agrawal) [inline igor-web/igor-web.gradle:102]: already exist in line 99

File: igor-web/igor-web.gradle [hunk 2]
```diff
@@ -4,93 +4,134 @@ mainClassName = 'com.netflix.spinnaker.igor.Main'
...
+  testImplementation "com.fasterxml.jackson.module:jackson-module-jaxb-annotations"
+  testImplementation "jakarta.activation:jakarta.activation-api:2.1.0"
+  testImplementation "org.springframework.cloud:spring-cloud-config-server"
+  implementation "com.squareup.okio:okio-jvm:3.2.0"
+//    testImplementation "javax.xml.bind:jaxb-api:2.3.1"
+//    testImplementation "com.sun.xml.bind:jaxb-core:4.0.0"
+//    testImplementation "com.sun.xml.bind:jaxb-impl:4.0.0"
+//    testImplementation "javax.activation:activation"
+//    testImplementation "jakarta.activation:jakarta.activation-api:2.1.0"
+//    testImplementation "jakarta.xml.bind:jakarta.xml.bind-api:4.0.0"
```
Comment (aman-agrawal) [inline igor-web/igor-web.gradle:124]: we can remove these commented dependencies (if we are not using)

**Comment by aman-agrawal** [comment]
Verified few commits from sheet - LGTM
Verified .gradle files - LGTM (just check few comments)

File: igor-web/igor-web.gradle (hunk 1 above)
Comment (emanipravallika) [reply igor-web/igor-web.gradle]: done. pushed the commit

File: igor-web/igor-web.gradle (hunk 2 above)
Comment (emanipravallika) [reply igor-web/igor-web.gradle]: done, pushed the commit.

**Review by j-sandy — APPROVED** [review]
LGTM !
@emanipravallika If first level review has been done and all the changes were taken care, then please go ahead for merge/closure of this PR

**Pull request merged**